lib/ioki/model/operator/geojson.rb

```diff
@@ -6,8 +6,8 @@ module Operator
       class Geojson < Base
         unvalidated true
 
-        attribute :coordinates, type: :array, on: :read
-        attribute :type, type: :string, on: :read
+        attribute :coordinates, type: :array, on: [:create, :read]
+        attribute :type, type: :string, on: [:create, :read]
```

Comment:
I don't think you need `type` on `:create` - at least I believe so :slightly_smiling_face:

Reply:
Different type here, so we apparently need that. Sorry for the noise!

Reply:
no problem 👍